openapi-maven-plugin/src/main/java/io/github/kbuntrock/yaml/YamlWriter.java

```diff
@@ -71,6 +71,8 @@ public class YamlWriter {
 	public YamlWriter(final MavenProject mavenProject, final ApiConfiguration apiConfiguration) {
 		this.apiConfiguration = apiConfiguration;
 		this.mavenProject = mavenProject;
+		this.om = apiConfiguration.getFileFormat() == FILEFORMAT_JSON ? new ObjectMapper() : new ObjectMapper(new YAMLFactory().enable(YAMLGenerator.Feature.MINIMIZE_QUOTES)
```

Comment:
I would prefer a "pretty print" json file. The single line is quite hard to read. Can you investigate on it? It should be just some configuration here.